app/search/styles.scss

```diff
@@ -86,3 +96,64 @@
     display: flex;
     justify-content: space-between;
 }
+
+div[role='tooltip'] {
```

Comment:
This selector may be causing issues globally, notably on the Overview page. A way to select and update the ember-tooltip div locally would be helpful.

Reply:
Yea, this rule will apply to _all_ divs that have `role=tooltip`, so this will ripple outside the search page. You can use the [`@tooltipClass`](https://github.com/sir-dunxalot/ember-tooltips#tooltipclass) and/or  [`@innerClass`](https://github.com/sir-dunxalot/ember-tooltips#innerclass) depending on what you want to style

Reply:
Futa, both of these do not select the element resulting in no styles applied:

<img width="1679" alt="image" src="https://github.com/CenterForOpenScience/ember-osf-web/assets/82047646/8de85965-70de-4eca-9141-a4e38c779e07">